lib/api/jobRouter.js

```diff
@@ -0,0 +1,36 @@
+const express = require("express");
+const router = express.Router();
+
+const jobController = require("../dbConnections/jobController");
+const { Status } = require("../constants/responseAndStatus");
+const logger = require("../logger/log");
+
+router.post("/create", (req, res) => {
+    if (!req.body) {
+        return res.status(400).json({
+            "error": "Bad Request"
+        });
+    } else {
+        jobController.createJob(req.body.id, req.body.description, req.body.frequency, req.body.script)
```

Comment:
While this is fine for the 1st cut, you should have a way to validate the "script" passed to the request body. This needs dynamic syntax checker to be run on the script supplied by the client